votes/0571-LittleFall-as-tidb-reviewer.md

```diff
@@ -0,0 +1,22 @@
+# A Vote for LittleFall as TiDB Reviewer
+
+## Proposal
+
+[@LittleFall](https://github.com/LittleFall) has been working on `pingcap/tidb` for several months. He has contributed a lot of bugfixes/enhancements for TiDB. His contributions covered a varity of modules of TiDB-Server(expression, executor, planner...). The details of his contributions are listed as follow:
+
+* [Authored pull requests](https://github.com/pingcap/tidb/commits?author=LittleFall)
+* [Reviewed pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+reviewed-by%3ALittleFall)
+
+I (@XuHuaiyu) hereby nominate @LittleFall as TiDB Reviewer and call for a vote.
+
+## Deadline
+
+The vote will be open for at least 6 days unless there is an objection or not enough votes.
```

Comment:
/hold

```suggestion
The vote will be open for at least 3 days unless there is an objection or not enough votes.
```

If you don't intend to use 6 days, our rules require at least 3 days - you can extend the requirement, but I guess it's not the purpose.

See also https://github.com/pingcap/community/blob/master/teams/README.md#actions